src/auth/credential.ts

```diff
@@ -419,6 +419,23 @@ export class ApplicationDefaultCredential implements FirebaseCredential {
   }
 }
 
+export class DatabaseEmulatorCredential implements FirebaseCredential {
+  public getAccessToken(): Promise<GoogleOAuthAccessToken> {
+    return Promise.resolve({
+      expires_in: 1000000,
+      access_token: 'owner',
+    });
+  }
+
+  public getCertificate(): Certificate {
```

Comment:
You should return `{}` (which is what the emulator does) or drop this entirely.  After https://github.com/firebase/firebase-admin-node/pull/576 I don't think it's needed.

Having stubbed values (`''`) will definitely cause more problems than it solves.

Reply:
The `FirebaseCredential` interface still requires this method, however 'null' is a valid return type according to its interface.

Reply:
#576 will remove this method from the interface.

Reply:
I think that PR removes it from the `Credential` interface, not the `FirebaseCredentialInterface`. Is there a reason to use the latter as opposed to the former? I just used it because it looks like that's the interface that the ApplicationDefaultCredential implements.

Reply:
Credential is the public interface. FirebaseCredential is for our internal use only. You should definitely be using the Credential interface.